README.md

```diff
@@ -44,13 +45,15 @@ git clone https://github.com/openware/opendax.git
 ```bash
 gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
 curl -sSL https://get.rvm.io | bash -s stable
+source /home/$USER/.rvm/scripts/rvm
```

Comment:
It's also a good idea to add this line to `~/.zshrc` or `~/.bashrc` to avoid running it for every session

Reply:
added to `~/.bashrc`